### PR TITLE
d4mac,win 17.06 edge release notes, docs updates

### PR DESCRIPTION
**Pull request by londoncalling**
### What's changed

- Release notes for Docker for Mac and Windows 17.06 Edge release (17.06.0-rc1-ce-mac13, 17.06.0-rc1-ce-win13)

- Documented new Edge features for Mac and added screen snaps
  - GUI Option to opt out of credential store
  - GUI option to reset docker data without losing all settings 

- Documented new Edge features on Windows and added screen snaps
  - Shared drive on demand 

- Cleaned up outdated mentions of "beta" specific items

- Expanded TOCs in the Getting Started topics for Mac and Windows to surface lower level topics (e.g., stuff on General, Advanced, and so forth was hidden in the on-page TOC's before)

### Reviewers

@friism @jeanlaurent @gtardif @dgageot @mstanleyjones @JimGalasyn 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

File: docker-for-mac/index.md [hunk 1]
```diff
@@ -131,6 +133,16 @@ You can set the following runtime options.
   Uncheck this to opt out and prevent auto-send of data. Docker may prompt for
   more information in some cases, even with auto-send enabled.
 
+#### Choose whether to store Docker credentials on Mac keychain (Edge feature)
```
Comment (mstanleyjones) [inline docker-for-mac/index.md]: There is way to call this out as an edge feature:

```
{% include edge_only.md section="feature" %}
```

It adds a note. So maybe put it right under the heading.

Comment (londoncalling) [inline docker-for-mac/index.md]: Can I keep my heading and just add that? The headings are surfaced in the on-page TOCs so I want to keep them

Comment (londoncalling) [inline docker-for-mac/index.md]: I also have some Edge feature notes that provide links and give more info that I don't want to replace. I'm testing this to see what it gives me.

Comment (londoncalling) [inline docker-for-mac/index.md]: @mstanleyjones Okay I tried that include and it weirdly had no effect (didn't show up). I'm going to let this finish building, and merge to get it published, but I'll experiment with those this afternoon. I have some other stuff to fix as well, later. Thanks!

Comment (mstanleyjones) [inline docker-for-mac/index.md]: NP :)

**Pull request merged**